composer.json

```diff
@@ -21,11 +21,11 @@
   },
   "require": {
     "php": "^5.6 || ^7.0",
-    "psr/http-message": "~1.0"
+    "psr/http-message": "^1.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^4.6 || ^5.5",
-    "zendframework/zend-coding-standard": "~1.0.0",
+    "phpunit/phpunit": "^5.6 || ^6.0",
```

Comment:
We need to get a bit more specific on these, as there's breakage in early versions of 6.0 and 5.6 that have often led to test failures in other repos. Try `^5.7.16 || ^6.0.8` here.